src/OnlineSales/Program.cs

```diff
@@ -50,6 +51,8 @@ public static void Main(string[] args)
 
         builder.Services.AddAutoMapper(typeof(Program));
 
+        builder.Services.AddHttpContextAccessor();
```

Comment:
Why is this accessor required here? What is the reason for it?

Reply:
@peterliapin  - Previously accessor has used in the Plugin project, now it has removed